### PR TITLE
Lra 497 lsa ride share add header footer static pages

### PR DESCRIPTION
**Pull request by britaumich**
A header with mobile menu, a footer with feedback are added.

### CODE REVIEW

File: app/assets/stylesheets/application.tailwind.css [hunk 1]
```diff
@@ -56,15 +56,15 @@ from https://tailwindcss.com/docs/adding-custom-styles
...
 
   .caption {
-    @apply text-sm font-semibold leading-4 italic text-gray-500;
+    @apply text-sm font-semibold leading-4 italic text-gray-550;
   }
 
   .breadcrumb {
-    @apply text-sm font-medium leading-4 text-gray-500;
+    @apply text-sm font-medium leading-4 text-gray-550;
   }
```
Comment (anantaUM) [inline app/assets/stylesheets/application.tailwind.css:69]: Why did we change the text color here ? Is it approved by Maria ?

**Review by anantaUM — COMMENTED** [review]
Reviewed and added a comment.

File: app/assets/stylesheets/application.tailwind.css (hunk 1 above)
Comment (britaumich) [reply app/assets/stylesheets/application.tailwind.css:69]: Yes, this is the color that Maria suggested in her Figma board. The link is in the ticket: https://umlsait.atlassian.net/browse/LRA-457

**Review by anantaUM — APPROVED** [review]

File: app/assets/stylesheets/application.tailwind.css (hunk 1 above)
Comment (anantaUM) [reply app/assets/stylesheets/application.tailwind.css:69]: Pull request approved.

**Review by rsmoke — APPROVED** [review]

**Pull request merged**